privacy_estimates/binomial_proportion.py

```diff
@@ -26,7 +25,8 @@ def compute_eps_lo_hi(count: AttackResults, delta: float, alpha: float, method:
             Default is 'jeffreys'
 
     Returns:
-        eps_lo, eps_hi (Tuple[float, float]): two-sided interval [eps_lo, eps_hi] with confidence 100*(1 - alpha)%.
+        fpr_l, fpr_r, fnr_l, fnr_r (Tuple[float, float, float, float]): 
```

Comment:
The order of the returned values is now swapped.

Reply:
Good catch, forgot to change the method annotations.